tests/rules/test_line_length.py

```diff
@@ -116,6 +116,16 @@ def test_non_breakable_word(self):
                    'long_line: http://localhost/very/very/long/url\n'
                    '...\n', conf, problem=(2, 21))
 
+        conf = 'line-length: {max: 20, allow-non-breakable-words: true}'
+        self.check('---\n'
+                   '# http://www.verylongurlurlurlurlurlurlurlurl.com\n'
+                   'key:\n'
+                   '  value:\n', conf)
+        self.check('---\n'
+                   '## http://www.verylongurlurlurlurlurlurlurlurl.com\n'
+                   'key:\n'
+                   '  value:\n', conf)
```

Comment:
Can you add these positive counterparts?
```python
        self.check('---\n'
                   '# # http://www.verylongurlurlurlurlurlurlurlurl.com\n'
                   'key:\n'
                   '  value:\n', conf, problem=(2, 21))
        self.check('---\n'
                   '#A http://www.verylongurlurlurlurlurlurlurlurl.com\n'
                   'key:\n'
                   '  value:\n', conf, problem=(2, 21))
```

Reply:
Done, please check